pkg/cloud/scope/session.go

```diff
@@ -33,11 +33,21 @@ func sessionForRegion(region string) (*session.Session, error) {
 		return s.(*session.Session), nil
 	}
 
-	ns, err := session.NewSession(aws.NewConfig().WithRegion(region))
+	ns, err := session.NewSessionWithOptions(session.Options{
```

Comment:
Would it be possible to consider the following modification below as an option to support credentials as an option? This would allow the user to specify session options in the bootstrap template until the full UX is developed.